Altis_Life.Altis/core/items/fn_boltcutter.sqf

```diff
@@ -60,45 +80,45 @@ switch (typeOf _building) do {
 
 for "_i" from 0 to 1 step 0 do {
     if (animationState player != "AinvPknlMstpSnonWnonDnon_medic_1") then {
-        [player,"AinvPknlMstpSnonWnonDnon_medic_1",true] remoteExecCall ["life_fnc_animSync",RCLIENT];
+        [player, "AinvPknlMstpSnonWnonDnon_medic_1", true] remoteExecCall ["life_fnc_animSync", RCLIENT];
         player switchMove "AinvPknlMstpSnonWnonDnon_medic_1";
         player playMoveNow "AinvPknlMstpSnonWnonDnon_medic_1";
     };
     sleep 0.26;
     if (isNull _ui) then {
-        "progressBar" cutRsc ["life_progress","PLAIN"];
+        "progressBar" cutRsc ["life_progress", "PLAIN"];
         _ui = uiNamespace getVariable "life_progress";
         _progressBar = _ui displayCtrl 38201;
         _titleText = _ui displayCtrl 38202;
     };
     _cP = _cP + _cpRate;
     _progressBar progressSetPosition _cP;
-    _titleText ctrlSetText format ["%3 (%1%2)...",round(_cP * 100),"%",_title];
+    _titleText ctrlSetText format ["%3 (%1%2)...", round(_cP * 100), "%", _title];
     if (_cP >= 1 || !alive player) exitWith {};
```

Comment:
See [this comment](https://github.com/AsYetUntitled/Framework/pull/73/files#r97928258).